test/functional/.gitignore

```diff
@@ -1,2 +1,4 @@
 plugins/
 results/
+config/dut_config.yml
+/config/duts_in_use
```

Comment:
Is 'config' and '/config' both expected and valid here?

Reply:
No, should be without backslash.